app.rb

```diff
@@ -0,0 +1,100 @@
+require_relative 'book'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'person'
+require_relative 'rental'
+
+def option1
+  puts '** Books **'
+  ObjectSpace.each_object(Book) do |obj|
+    puts "Title: #{obj.title} Author: #{obj.author}"
+  end
+end
+
+def option2
+  puts '** Teachers **'
+  ObjectSpace.each_object(Teacher) do |obj|
+    puts "Id: #{obj.id} Name: #{obj.name} Age: #{obj.age} Specialization: #{obj.specialization}"
+  end
+  puts '** Students **'
+  ObjectSpace.each_object(Student) do |obj|
+    puts "Id: #{obj.id} Name: #{obj.name} Age: #{obj.age}"
+  end
+end
```

Comment:
- [ ] I also noticed that, when I tried to `List all people` using option_2, another error was thrown and as a result, I couldn't see the list of teachers that I created. So kindly fix that as well :100: 
- [ ] I also noticed that the details shown to a user when trying to create a rental aren't similar to what was shown in the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) as a result, I was unable to access the ID of the person who created the rental so I was unable to view the list of rentals for a given person. So kindly fix that as well :100: 
- Example Video

[Screencast from 07-01-2023 01:06:42 PM.webm](https://github.com/haftamudesta/school-library/assets/102699958/fdec1b1b-98e7-4e4a-9f40-58bcb0bfd711)